AbstractSDK/abstract
# Diff PR next release

- Changed account id structure
- Corrected tests
- formatting
- Cleanup and test
- Changelog
- Corrected imports
- Fixed tests
- improve ChainName construction strictness
- fix tests
- Cleanup and storage key shortening
- fix test
- clippy fix
- remove osmosis remenant
- bump cw-orch
- Hot fix for cw-orch 0.16.0
- Moved from_str to trait
- bump cw-orch
- fix tests
- formatting
- bump version to 0.19
- formatting
- Update deploy_all bin (#75)
- add release
- update yml
- add job
- add new release action
- updates circleci
- update config
- add when, add deploy circle ci req
- edit ci config
- remove graph png
- test release branch match
- update deploy branch match
- poll circleci api status
- Update WASM artifacts [skip ci]
- fetch workflow_id
- check endpoint get workflow id for status
- force wasm diff commit with comment
- fix parse response bug
- check for null values on parse, log if any
- add more logs for parse
- add wait incase of propogation delay
- update config ci
- handle all status cases from ci api res
- Revert "update config ci"
- update status handle
- update config ci after revert
- add logs
- change item to check on response
- use pipeline id for prev pipeline qry
- will update checksum, force rebuild wasm
- update bin to force wasm write
- force wasm commit
- add arbitary to force wasm diff
- add post on success
- remove syntax bash error
- Revert "force wasm commit"
- remove polling logic
- edit release run command
- update config ci
- add new release workflow
- change metadata req
- update req
- typo config
- testing ci
- curl
- log response
- fetch branch name
- remove binaries
- remove binaries
- Update WASM artifacts [skip ci]
- add artifcats dir
- remove bins (#84)
- Update WASM artifacts [skip ci]
- merge main
- update gitignore
- Update WASM artifacts [skip ci]
- skip build, add path
- update config
- update config
- add requries back
- update param namespace
- Update WASM artifacts [skip ci]
- add artifacts folder inside framework/
- remove binaries
- skip build step
- set log level
- will trigger build
- Update WASM artifacts [skip ci]
- set env var for container
- skip build
- add log var
- echo container vars
- log env
- set var in docker
- fix typo in set command
- echo env var
- remove override
- decode base64 env var
- decode base64
- combine commands to persist env state
- log
- remove log
- add commit deploy file step
- Update deploy deploy file [skip ci]
- streamline publishing process
- formatting
- commit push cw-orch state file
- remove binaries for commit
- add build step back
- Update WASM artifacts [skip ci]
- Revert "formatting"
- add check for wasms, skip if no diff
- skip build for now
- format yml
- remove first deploy
- remove check
- add manual wasm build action
- remove redundant comment
- formatting
- update changelog
- use branch parameter for build
- update manual build
- formatting
- use branch env var
- Update WASM binaries [skip ci]
- deploy abstract
- formatting
- harpoon-4 deploy
- 0.19.1 abstract-interface bump
- fix clippy
- fix and publish standards
- update publish
- formatting
- Corrected cw-plus
- formatting
- fix thiserror
- bump thiserror
- bump abstract-app and abstract-adapter
- bump all versions to 0.19.2
- remove thiserror version constraint
- Update WASM binaries [skip ci]
- formatting
- remove path deps from integrations
- formatting
- Add kujira testnet, sei, neutron mainnet, and nibiru testnet deployments
- Update 0.19 schemas
- Add Archway 0.19 testnet deployment
- Update archway testnet deployment addrs and add doravota
- Add archway mainnet deployment

### Checklist
